backend/src/main/java/com/pickpick/config/ClockConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.pickpick.config;
+
+import java.time.Clock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ClockConfig {
+
+    @Bean
+    public Clock clock() {
+        return Clock.systemDefaultZone();
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#systemDefaultZone--

systemDefault의 의미가 무엇을지 궁금해서 찾아봤는데요, 내부 코드를 봐야 조금 더 알 수 있울 것 같네요. 주석만 봐선 best available system clock이 의미하는 바를 이해하기 어렵네요. 다만 아래와 같은 내용은 참고해볼만한 것 같아 공유합니다.

가능하면 타임존을 설정해서 사용하길 권장하는 것 같습니다.

Using this method hard codes a dependency to the default time-zone into your application. It is recommended to avoid this and use a specific time-zone whenever possible. 

Reply:
저희가 TimeZoneConfig 통해서 TimeZone을 세팅하고 있기 때문에 여기선 별도로 설정하지 않았는데요🤔
만약에 TimeZone을 변경할 일이 생기면 변경 포인트가 불필요하게 늘어난다는 생각이 들었어요!

ClockConfig와 TimeZoneConfig를 TimeConfig 하나로 합치는 방법도 있을 것 같은데 어떤게 더 좋을지 함께 고민해주시면 감사합니다 😄

Reply:
시간 관련 설정을 하나의 클래스에서 관리하는 게 좋을 것 같습니다

저는 현재 저희 서비스에선 Asia/Seoul 이외의 지역으로
TimeZone을 변경할 경우를 고려하지 않아도 된다고 생각합니다만
변경할 일에 대해서 고려한다면 더욱 시간 관련된 설정이 한 곳에 모여있는 게 좋을 것 같습니다.

systemDefault라는 것이 setDefault를 설정하는 클래스내부에 응집도 있게 모여있다면
맥락상 해석이 가능하겠지만
ClockConfig만 보아서는 직접개발했던 사람이 아니라면 맥락이 분산된 상황인 것 같습니다